tests/TestCodeGen.hs

```diff
@@ -51,10 +51,11 @@ pythonInteroperation = testGroup "Python interoperation" $ do
 #else
   recStyle <- [RegularRecords]
 #endif
+  isPrefixedArg <- [IsPrefixed True, IsPrefixed False]
```

Comment:
Comment only: this will make these tests take twice as long, and they are already lengthy, but I think it is still acceptable.  If that changes, we can consider random exploration of the configuration space.